From 377039d24648f82dac35dcf22a2b43de81f2fb96 Mon Sep 17 00:00:00 2001
From: Petr Mensik <pemensik@redhat.com>
Date: Thu, 11 Nov 2021 05:48:38 +0100
Subject: [PATCH] Skip test case, which always hangs

It just stops and does not continue. Avoid that test.
---
 src/libtls/tests/suites/test_socket.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/libtls/tests/suites/test_socket.c b/src/libtls/tests/suites/test_socket.c
index 9e26e91..5296680 100644
--- a/src/libtls/tests/suites/test_socket.c
+++ b/src/libtls/tests/suites/test_socket.c
@@ -804,11 +804,13 @@ Suite *socket_suite_create()
 	add_tls_versions_test(test_tls_12_server, TLS_1_0, TLS_1_3);
 	suite_add_tcase(s, tc);
 
+#if 0
 	tc = tcase_create("TLS 1.3/key exchange groups");
 	tcase_add_checked_fixture(tc, setup_creds, teardown_creds);
 	tcase_add_loop_test(tc, test_tls13_ke_groups, 0,
 						tls_crypto_get_supported_groups(NULL));
 	suite_add_tcase(s, tc);
+#endif
 
 	tc = tcase_create("TLS 1.3/signature schemes");
 	tcase_add_checked_fixture(tc, setup_all_creds, teardown_creds);
-- 
2.31.1

